test_schemas/en/test_new_routing_to_section_end.json

```diff
@@ -0,0 +1,200 @@
+{
```

Comment:
If the other new schemas that have been added route to the section end, why is this schema needed?

Reply:
That was needed when loop existed, after adding changes from your comments this is redundant and removed, thanks!